build.gradle

```diff
@@ -5,6 +5,7 @@ plugins {
 }
 
 repositories {
+    mavenLocal()
```

Comment:
Not sure to understand why this is required but I supposed this is kind of standard

Reply:
@jeffmaury I consider this as useful because it allows me publish to the local repo and consume test in consuming projects. That's how we proceed in maven. Missing something?